eos-phone-home

```diff
@@ -273,6 +285,54 @@ class PhoneHome(object):
 
         return False
 
+    def _get_mac_hash(self):
+        try:
+            ifaces = []
+            wireless = []
+
+            for iface in glob.glob('/sys/class/net/*'):
```

Comment:
As we talked about it on Slack, maybe this whole thing would be much simpler with `uuid.getnode()`